modules/core/src/main/scala/org/tessellation/http/routes/GlobalSnapshotRoutes.scala

```diff
@@ -0,0 +1,28 @@
+package org.tessellation.http.routes
+
+import cats.Monad
+
+import org.tessellation.domain.snapshot.GlobalSnapshotStorage
+import org.tessellation.ext.http4s.vars.SnapshotOrdinalVar
+
+import org.http4s.HttpRoutes
+import org.http4s.circe.CirceEntityCodec.circeEntityEncoder
+import org.http4s.dsl.Http4sDsl
+import org.http4s.server.Router
+
+final case class GlobalSnapshotRoutes[F[_]: Monad](globalSnapshotStorage: GlobalSnapshotStorage[F])
+    extends Http4sDsl[F] {
+  private val prefixPath = "/global-snapshot"
+
+  private val httpRoutes: HttpRoutes[F] = HttpRoutes.of[F] {
+    case GET -> Root / "latest" =>
+      Ok(globalSnapshotStorage.getLast)
```

Comment:
To be consistent maybe it should be either "/latest" and ".getLatest" or "/last" and ".getLast"?